bin/finalize_microbiome_entities.py

```diff
@@ -61,26 +62,36 @@ def process_weights(subset):
 
 if not args.entrez_microbiomes_entities and not args.nucl_microbiomes_entities:
     sys.exit("Neither --entrez-microbiome-entities nor --nucl-microbiome-entities were specified. Aborting.")
+    # What about input type "proteins"?
 
 # Read and join the tables that provide microbiome_id, entity_id and entity_name
 entity_microbiome = pd.read_csv(args.microbiomes_entities_noweights, sep='\t')
 entity            = pd.read_csv(args.entities, sep='\t')
+conditions        = pd.read_csv(args.conditions, sep="\t")
 
-entity_microbiome = entity_microbiome.merge(entity)
+entity_condition = entity_microbiome.merge(entity).merge(conditions)
```

Comment:
I agree this script is confusing, this is because microbiome_entities.entrez.tsv uses microbiome_ids, while microbiomes_entities.nucl.tsv uses condition_id. I didn't rename them in order for us to be able to connect them to what they used to be, but I guess that is much more confusing, I'll try to find appropriate names. The microbiome_id here is used later for connecting the entities from entrez to the condition_id.